pkgs/by-name/gi/gitSetupHook/gitSetupHook.sh

```diff
@@ -0,0 +1,12 @@
+# shellcheck shell=bash
+# This script is a hook that sets up a minimal git configuration
+
+gitSetup () {
+    echo "Setting up minimal git configuration..."
```

Comment:
Nor this tbh, but we definitely are inconsistent in this area.

Reply:
Removed.